lib/config-generator.js

```diff
@@ -298,6 +298,14 @@ export default [\n${exportContent}];`;
             } else {
                 log.info(`Successfully created ${configPath} file.`);
             }
+
+            if (packageManager === "npm" && this.result.installFlags.includes("--force")) {
+                const pkg = JSON.parse(await path.readFile(this.packageJsonPath, "utf8"));
+
+                log.warn("Note some plugins currently do not support ESLint v9 yet.\nYou will need to use '--force' when installing, or add the following to your package.json:\n");
```

Comment:
Shouldn't this come in the `else` block below? We'd want to log this only when users need to install the dependencies themselves.

Reply:
I believe this was intentional. The problem is not when `@eslint/create-config` installs dependencies, but when the user runs `npm install` anytime after that.

Reply:
Then it should print warning in both the cases right?

Reply:
Yeah, that would definitely make sense, but the problem is that in the other case we don't know if it's needed because we don't know which package manager is used in the project. Perhaps we could add something like `(not needed if you are using yarn`).

Reply:
well, I adjusted the message:

when choosing npm:
```bash
Note some plugins currently do not support ESLint v9 yet.
You may need to use '--force' when installing, or add the following to your package.json: 
"overrides": { "eslint": "^9.5.0" } 
```
others:
```bash
Note some plugins currently do not support ESLint v9 yet.
You may need to use '--force' when installing
```
I'm not a native English speaker, so suggestions for wording improvements are welcome! 😄

Reply:
> we don't know which package manager is used in the project.

We can identify which package manager is being used by checking for lock files? 


> Note some plugins currently do not support ESLint v9 yet.

this can be changed to `Note that some plugins currently do not support ESLint v9 yet.`

Reply:
> > We can identify which package manager is being used by checking for lock files?
> 
> maybe, I'm just not sure it's worthy the efforts - the plugins will add official eslint v9 supports later. :)

I also think it's not worth the effort at this point, both plugins will probably add official eslint v9 support soon.